### ISSUE TITLE
Different search providers

### ISSUE THREAD

**Issue by astra-midnight** [spectresystems/jarvis#76]
I use DuckDuckGo and not Google so being able to switch search providers would be a nice addition.

**Comment by ocdtrekkie** [comment]
Indeed. Web search being Google-only would absolutely be a non-starter for me on using this tool.
